python/src/opendp/extras/polars/__init__.py

```diff
@@ -410,6 +410,7 @@ def _discrete_quantile_score(self, alpha: float, candidates: list[float]):
             function_name="discrete_quantile_score",
             args=[self.expr, alpha, Series(candidates)],
             returns_scalar=True,
+            changes_length=True,
```

Comment:
Not obvious that an upgrade would require a change here. Double-check that it's not stray, but no particular concerns beyond that.

Reply:
This now segfaults without it. Was a bit tricky to track down.